src/main/java/edu/wpi/first/pathweaver/PathDisplayController.java

```diff
@@ -28,7 +28,7 @@
   private Pane topPane;
   private final PseudoClass selected = PseudoClass.getPseudoClass("selected");
   private Waypoint selectedWaypoint = null;
-  private final ObjectProperty<Path> currentPath = new SimpleObjectProperty<>();
+  public final ObjectProperty<Path> currentPath = new SimpleObjectProperty<>();
```

Comment:
JavaFX paradigm for this is private field aproperty with a method named aproperty() to get it.  Let's do that instead.